### PR TITLE
Started kickoff of the project

### PR DESCRIPTION
**Pull request by nvescio**
Summary
 - Project setup
 - Page Titles
 - Technology

Screenshots
Chrome
maquetado1

Trello Card
 - https://trello.com/c/Pj5i7OZQ/11-setup-del-proyecto
 - https://trello.com/c/jrAwRI7I/21-page-titles
 - https://trello.com/c/TIs5kuHC/22-tecnolog%C3%ADa

### CODE REVIEW

File: training_maquetado/index.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  <div class="tech-container"> 
+        <img class="tech-icon" src="assets/react_icon.svg">
+        <h2 class="tech-title">ReactJS</h2>
+        <h2 class="tech-release">Release: 2013</h2>
+        <p class="tech-desc">React is a declarative, component-based JavaScript library for building user interfaces.</p>
+  </div>   
+
+</body>
+
+</html>
```
Comment (CarlosHWolox) [inline training_maquetado/index.html]: We usually leave an empty line at the end of the file so this icon do not appear on github

File: training_maquetado/index.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+</head>
+
+<body>
+  <div class="title-container"> 
+    <img class="triangle" src="assets/triangle.svg">
```
Comment (CarlosHWolox) [inline training_maquetado/index.html]: Use a more descriptive name for classes

File: training_maquetado/scss/titles.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
+.title-container {
+    display:flex;
```
Comment (CarlosHWolox) [inline training_maquetado/scss/titles.scss]: space

File: training_maquetado/scss/titles.scss [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    align-items: center;
+}
+
+.title {
+    color: black;
+    font-family: $roboto;
+    font-size: $text-big;
+    position: absolute;
+    padding-left: $title-decoration-width - 20px;
```
Comment (CarlosHWolox) [inline training_maquetado/scss/titles.scss]: i think this variable here is unnecesary

File: training_maquetado/.gitignore [hunk 5]
```diff
@@ -0,0 +1,10 @@
+### Sass ###
+.sass-cache/
+.scss-cache/
+*.css.map
+*.sass.map
+*.scss.map
+*.css
+
+## System ##
+.DS_Store
```
Comment (FrankIglesias) [inline training_maquetado/.gitignore]: add tailing line

File: training_maquetado/scss/colors.scss [hunk 6]
```diff
@@ -0,0 +1,3 @@
+$blue: #00ADEE;
+$green: #BED23C;
+$grey: #C4C4C4;
```
Comment (FrankIglesias) [inline training_maquetado/scss/colors.scss]: same

File: training_maquetado/scss/equalizer.scss [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+::placeholder {
+  color: $equalizer-grey;
+  opacity: 1;
+}
+
```
Comment (FrankIglesias) [inline training_maquetado/scss/equalizer.scss]: delete extra spaces

File: training_maquetado/scss/tech.scss [hunk 8]
```diff
@@ -0,0 +1,51 @@
+.tech-container {
```
Comment (FrankIglesias) [inline training_maquetado/scss/tech.scss:1]: indentation

File: training_maquetado/scss/tech.scss [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+.tech-release {
+    font-weight: bold;
+}
+
+.tech-desc {
+    margin: 12px 0;
+}
+
```
Comment (FrankIglesias) [inline training_maquetado/scss/tech.scss]: delete this two empty lines

File: training_maquetado/scss/titles.scss [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+} 
+
+.triangle {
```
Comment (FrankIglesias) [inline training_maquetado/scss/titles.scss]: check indentation

File: training_maquetado/scss/titles.scss [hunk 11]
```diff
@@ -0,0 +1,17 @@
...
+    height: $title-decoration-height;
```
Comment (FrankIglesias) [inline training_maquetado/scss/titles.scss]: set only width as fixed

File: training_maquetado/index.html [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+</head>
+
```
Comment (FrankIglesias) [inline training_maquetado/index.html]: delete line

File: training_maquetado/index.html [hunk 13]
```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
```
Comment (FrankIglesias) [inline training_maquetado/index.html]: change this one for roboto

File: training_maquetado/index.html [hunk 14]
```diff
@@ -0,0 +1,28 @@
...
+    <h1 class="title">Front-End Tools</h1>
+  </div> 
+
+  <div class="tech-container"> 
+        <img class="tech-icon" src="assets/react_icon.svg">
+        <h2 class="tech-title">ReactJS</h2>
+        <h2 class="tech-release">Release: 2013</h2>
+        <p class="tech-desc">React is a declarative, component-based JavaScript library for building user interfaces.</p>
+  </div>   
+
```
Comment (FrankIglesias) [inline training_maquetado/index.html]: delete line

File: training_maquetado/index.html [hunk 15]
```diff
@@ -0,0 +1,28 @@
...
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+</head>
+
+<body>
+  <div class="title-container">
```
Comment (FrankIglesias) [inline training_maquetado/index.html]: remove this div  and change it for a h1 instead

File: training_maquetado/index.html (hunk 2 above)
Comment (FrankIglesias) [inline training_maquetado/index.html]: add an alt

File: training_maquetado/index.html [hunk 16]
```diff
@@ -0,0 +1,28 @@
...
+  <link rel="stylesheet"  type="text/css" href="application.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+</head>
+
+<body>
+  <div class="title-container"> 
+    <img class="triangle" src="assets/triangle.svg">
+    <h1 class="title">Front-End Tools</h1>
+  </div> 
+
```
Comment (FrankIglesias) [inline training_maquetado/index.html]: remove this line

File: training_maquetado/index.html [hunk 17]
```diff
@@ -0,0 +1,28 @@
...
+  <div class="tech-container">
```
Comment (FrankIglesias) [inline training_maquetado/index.html:14]: check indentation

File: training_maquetado/index.html [hunk 18]
```diff
@@ -0,0 +1,28 @@
...
+</head>
+
+<body>
+  <div class="title-container"> 
+    <img class="triangle" src="assets/triangle.svg">
+    <h1 class="title">Front-End Tools</h1>
+  </div> 
+
+  <div class="tech-container"> 
+        <img class="tech-icon" src="assets/react_icon.svg">
```
Comment (FrankIglesias) [inline training_maquetado/index.html]: add an alt

File: training_maquetado/scss/styles.scss [hunk 19]
```diff
@@ -0,0 +1,10 @@
+$roboto: 'Open Sans', sans-serif;
+
+$nav-height: 70px;
+$nav-logo-height: 30px;
+$title-decoration-height: 90px;
+$title-decoration-width: 70px;
+$tech-logo-size: 100px;
+$text-big: 24px;
+$text-medium: 16px;
+$text-small: 12px;
```
Comment (FrankIglesias) [inline training_maquetado/scss/styles.scss]: add empty line at the bottom of the file

**Review by CarlosHWolox — APPROVED** [review]

File: training_maquetado/scss/styles.scss [hunk 20]
```diff
@@ -0,0 +1,10 @@
+$roboto: 'Roboto', sans-serif;
```
Comment (FrankIglesias) [inline training_maquetado/scss/styles.scss]: are you using this variable?

File: training_maquetado/scss/tech.scss [hunk 21]
```diff
@@ -0,0 +1,49 @@
...
+    display: flex;
+    align-items: center;
+    background-color: $green;
+    height: 300px;
+    flex-direction: column;
+    position: relative;
+    width: 200px;
+}
+
+.tech-container,
```
Comment (FrankIglesias) [inline training_maquetado/scss/tech.scss]: I'd do something like
```scss 
.tech-container {
...

  &::before {
     ....
   }
```

File: training_maquetado/scss/tech.scss [hunk 22]
```diff
@@ -0,0 +1,49 @@
...
+.tech-container:before {
+    border-radius: 10px;
+    box-sizing: border-box;
+}
+
+.tech-container:before {
```
Comment (FrankIglesias) [inline training_maquetado/scss/tech.scss]: join this with the `&::before`

File: training_maquetado/scss/tech.scss [hunk 23]
```diff
@@ -0,0 +1,49 @@
...
+    border: 1px solid $green;
+    content: "";
+    display: block;
```
Comment (FrankIglesias) [inline training_maquetado/scss/tech.scss]: this is unnecessary

File: training_maquetado/scss/tech.scss [hunk 24]
```diff
@@ -0,0 +1,49 @@
...
+    height: 100%;
+    left: -10px;
+    position: absolute;
```
Comment (FrankIglesias) [inline training_maquetado/scss/tech.scss]: this is not neccesary

File: training_maquetado/scss/tech.scss [hunk 25]
```diff
@@ -0,0 +1,49 @@
...
+    top: -10px;
+    width: 100%;
+    z-index: -1;
+}
+
+.tech-icon {
```
Comment (FrankIglesias) [inline training_maquetado/scss/tech.scss:28]: check indentation

File: training_maquetado/scss/tech.scss [hunk 26]
```diff
@@ -0,0 +1,49 @@
...
+    height: $tech-logo-size;
+    margin: 4px 0;
+    width: $tech-logo-size;
```
Comment (FrankIglesias) [inline training_maquetado/scss/tech.scss]: fix only width or margin not both

File: training_maquetado/scss/titles.scss [hunk 27]
```diff
@@ -0,0 +1,17 @@
+.title-container {
+    display: flex;
+    align-items: center;
+}
+
+.title {
+    color: black;
+    font-family: $roboto;
+    font-size: $text-big;
+    position: absolute;
```
Comment (FrankIglesias) [inline training_maquetado/scss/titles.scss]: You dont need this

File: training_maquetado/scss/titles.scss [hunk 28]
```diff
@@ -0,0 +1,17 @@
+.title-container {
+    display: flex;
+    align-items: center;
+}
+
+.title {
+    color: black;
+    font-family: $roboto;
```
Comment (FrankIglesias) [inline training_maquetado/scss/titles.scss]: move this prop to an upper element I'd use `*` or `body`

File: training_maquetado/index.html [hunk 29]
```diff
@@ -0,0 +1,28 @@
...
+<body>
+  <div class="title-container"> 
+    <img class="triangle-decorator" src="assets/triangle.svg">
+    <h1 class="title">Front-End Tools</h1>
+  </div> 
+
+  <div class="tech-container"> 
+        <img class="tech-icon" src="assets/react_icon.svg">
+        <h2 class="tech-title">ReactJS</h2>
+        <h2 class="tech-release">Release: 2013</h2>
```
Comment (FrankIglesias) [inline training_maquetado/index.html]: this should be a less important headline like 3 or 4

File: training_maquetado/scss/styles.scss (hunk 20 above)
Comment (FrankIglesias) [reply training_maquetado/scss/styles.scss]: This file should be renamed for constants

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: training_maquetado/scss/titles.scss [hunk 30]
```diff
@@ -0,0 +1,17 @@
+*{
+  font-family: $roboto;
+}
+
+.title {
+  align-items: center;
+  display: flex;
+  font-size: $text-big;
+  height: $title-decoration-height;
+    &::before {
```
Comment (FrankIglesias) [inline training_maquetado/scss/titles.scss:11]: leave an empty line  before the `&::before`

File: training_maquetado/scss/tech.scss [hunk 31]
```diff
@@ -0,0 +1,45 @@
...
+      border: 1px solid $green;
+      content: "";
+      height: 100%;
+      left: -10px;
+      position: absolute;
+      top: -10px;
+      width: 100%;
+      z-index: -1;
+    }
+    &,
```
Comment (FrankIglesias) [inline training_maquetado/scss/tech.scss:21]: same

File: training_maquetado/scss/tech.scss [hunk 32]
```diff
@@ -0,0 +1,45 @@
+.tech-container {
+  align-items: center;
+  background-color: $green;
+  display: flex;
+  flex-direction: column;
+  height: 300px;
+  position: relative;
+  width: 200px;
+    &::before {
```
Comment (FrankIglesias) [inline training_maquetado/scss/tech.scss:10]: same

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**